command/genesis/genesis.go

```diff
@@ -206,6 +227,8 @@ func (c *GenesisCommand) Run(args []string) int {
 	flags.Uint64Var(&epochSize, "epoch-size", ibft.DefaultEpochSize, "")
 	flags.Uint64Var(&blockGasLimit, "block-gas-limit", helper.GenesisGasLimit, "")
 	flags.BoolVar(&isPos, "pos", false, "")
+	flags.UintVar(&minNumValidator, "min-validator-count", 1, "")
+	flags.UintVar(&maxNumValidator, "max-validator-count", math.MaxUint32, "")
```

Comment:
These default values should be extracted out, since you use them in multiple places 👍

Reply:
Added in https://github.com/0xPolygon/polygon-edge/pull/397/commits/fe34bdd49d2f6aae469ffc12ae67d33f89fe280f